cgt_calc/parsers/mssb.py

```diff
@@ -0,0 +1,165 @@
+"""Morgan Stanley parser.
+
+Note, that I only had access to an Alphabet export. I have no idea how it looks like
+for another company, or for a full profile.
+"""
+from __future__ import annotations
+
+import csv
+from datetime import datetime
+from decimal import Decimal
+from pathlib import Path
+from typing import Final
+
+from cgt_calc.exceptions import ParsingError, UnexpectedColumnCountError
+from cgt_calc.model import ActionType, BrokerTransaction
+
+COLUMNS_RELEASE: Final[list[str]] = [
+    "Vest Date",
+    "Order Number",
+    "Plan",
+    "Type",
+    "Status",
+    "Price",
+    "Quantity",
+    "Net Cash Proceeds",
+    "Net Share Proceeds",
+    "Tax Payment Method",
+]
+
+COLUMNS_WITHDRAWAL: Final[list[str]] = [
+    "Date",
+    "Order Number",
+    "Plan",
+    "Type",
+    "Order Status",
+    "Price",
+    "Quantity",
+    "Net Amount",
+    "Net Share Proceeds",
+    "Tax Payment Method",
+]
+
+# These can be potentially wired through as a flag
+KNOWN_SYMBOL_DICT: Final[dict[str, str]] = {
+    "GSU Class C": "GOOG",
+}
+
+
+def __hacky_parse_decimal(decimal: str) -> Decimal:
```

Comment:
2 underscores are usually used for some python special methods. If you want to mark it private, I think single underscore is fine.
```suggestion
def _hacky_parse_decimal(decimal: str) -> Decimal:
```

Reply:
Done.